aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/layout/ClassicWindow.get.js

```diff
@@ -0,0 +1,40 @@
+model.jsonModel = {
+   services: [
+      "alfresco/services/ErrorReporter"
+   ],
+   widgets:[
+      {
+         id: "WINDOW1",
+         name: "alfresco/layout/ClassicWindow",
+         config: {
+            title: "Test title",
+            widgets: [
+               {
+                  name: "alfresco/logo/Logo",
+                  config: {
+                     logoClasses: "alfresco-logo-grey"
+                  }
+               }
+            ]
+         }
+      },
+      {
+         id: "WINDOW2",
+         name: "alfresco/layout/ClassicWindow",
+         config: {
+            hideTitle: true,
+            widgets: [
+               {
+                  name: "alfresco/logo/Logo",
+                  config: {
+                     logoClasses: "alfresco-logo-grey"
+                  }
+               }
+            ]
+         }
+      },
+      {
+         name: "aikauTesting/TestCoverageResults"
```

Comment:
This widget is no longer used for test coverage it can be removed, we've been removing them as we touch any existing WebScript test page, so we'd prefer not to add any more (also, unless I mention it this time they'll continue to get added).